.github/workflows/on-dependabot-pr.yml

```diff
@@ -0,0 +1,26 @@
+name: 'On Dependabot PR'
+
+# Ensures that only one workflow is run per branch at a time.
+concurrency:
+  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
+
+on: pull_request
+
+permissions:
+  pull-requests: write
+
+jobs:
+  dependabot-labels:
+    if: github.event.pull_request.user.login == 'dependabot[bot]' && github.repository == 'kumahq/kuma-gui'
```

Comment:
Is this not supposed to be synced up to KM?

Reply:
🤔 not even sure why I added that, let me look. I might have just copied it from the example 😱 

I can't think of a reason why we'd need to say "only this repository"

---

I just checked I think it came in with my copy paste of the example, let me remove it.